modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-repo/src/test/java/org/nuxeo/apidoc/test/RuntimeSnaphotRepoFeature.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * (C) Copyright 2012-2013 Nuxeo SA (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.nuxeo.apidoc.test;
+
+import org.nuxeo.ecm.automation.test.AutomationFeature;
+import org.nuxeo.ecm.core.test.CoreFeature;
+import org.nuxeo.ecm.core.test.annotations.Granularity;
+import org.nuxeo.ecm.core.test.annotations.RepositoryConfig;
+import org.nuxeo.elasticsearch.test.RepositoryElasticSearchFeature;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.RunnerFeature;
+
+/**
+ * @since 8.3
+ */
+@Features({ CoreFeature.class, RepositoryElasticSearchFeature.class, AutomationFeature.class })
+@RepositoryConfig(cleanup = Granularity.METHOD)
+@Deploy("org.nuxeo.apidoc.core")
+@Deploy("org.nuxeo.apidoc.repo")
+@Deploy("org.nuxeo.apidoc.repo:apidoc-works-test-contrib.xml")
+public class RuntimeSnaphotRepoFeature implements RunnerFeature {
```

Comment:
indeed, thx -- and actually: i created the "non repo" one since the startup is faster, but there are missing extension points when running it --> do we have any good practices to follow in that kind of cases please?

Reply:
gave up on this change anyway: i need the repo in all tests